test/transcoding/FPGALoopAttr.ll

```diff
@@ -23,7 +23,7 @@ entry:
   store i32 0, i32* %i, align 4
   br label %for.cond
 ; CHECK-SPIRV: 4 LoopMerge {{[0-9]+}} {{[0-9]+}} 4
```

Comment:
Please add a comment explaining which metadata was used in translation to get the LoopMerge argument(s).

Reply:
Would adding a comment with the LoopControlMask field(s) name(s) suffice? Akin to what I've done for `FPGAIVDepLoopAttr.ll`.

Reply:
I've updated the files with more comments - please decide if these are good enough.